docs/rule_groups/whitespace_rule_group.rst

```diff
@@ -150,3 +150,4 @@ Rules Enforcing Whitespace Rule Group
 * `whitespace_010 <../whitespace_rules.html#whitespace-010>`_
 * `whitespace_011 <../whitespace_rules.html#whitespace-011>`_
 * `whitespace_013 <../whitespace_rules.html#whitespace-013>`_
+* `whitespace_100 <../whitespace_rules.html#whitespace-013>`_
```

Comment:
@jeremiah-c-leary FYI, link on this line is incorrect.

Reply:
Good catch.  As I have already deleted the branch, I made the change on master.  You should see the link working in the latest version of the documentation.

--Jeremy